internal/librariangen/postprocessor/postprocessor_test.go

```diff
@@ -122,17 +173,47 @@ func TestPostProcess(t *testing.T) {
 				t.Fatalf("PostProcess() error = %v, wantErr %v", err, tt.wantErr)
 			}
 
+			if tt.singleNewAPI && !tt.wantErr {
+				if !goModInitCalled {
+					t.Error("go mod init was not called")
+				}
+				if !goModTidyCalled {
+					t.Error("go mod tidy was not called")
+				}
+			}
+			if !tt.singleNewAPI {
+				if goModInitCalled {
+					t.Error("go mod init was called unexpectedly")
+				}
+				if goModTidyCalled {
+					t.Error("go mod tidy was called unexpectedly")
+				}
+			}
+
 			if tt.wantErr {
 				return
 			}
 
+			// Determine which files should have been modified based on the request.
+			wantModifiedFiles := make(map[string]bool)
+			for _, api := range req.APIs {
```

Comment:
I'm *tempted* to suggest we should just have these listed in the test case, to avoid having logic that's effectively duplicating the production code here. I can see it would be tedious though. Hmm.